Dockerfile.run

```diff
@@ -0,0 +1,96 @@
+# Spreed WebRTC server in minimal Docker (for production)
+#
+# This Dockerfile creates a container which builds Spreed WebRTC as piped in
+# on stdin using another Docker container defined in `Dockerfile.build`.
+#
+# First create the builder image:
+#
+#   ```
+#   docker build -t spreed-webrtc-builder -f Dockerfile.build .
+#   ```
+#
+# Next run the builder container, piping its output into the creation of the
+# runner container:
+#
+#   ```
+#   docker run --rm spreed-webrtc-builder | docker build -t spreed-webrtc -f Dockerfile.run -
+#   ```
+#
+# image. Afterwards run the container like this:
+#
+#   ```
+#   docker run --rm --name my-spreed-webrtc -p 8080:8080 -p 8443:8443 \
+#       -v `pwd`:/srv/extra -i -t spreed-webrtc
+#   ```
+#
+# Now you can either use a frontend proxy like Nginx to provide TLS to Spreed
+# WebRTC and even run it in production like that from the Docker container, or
+# for easy development testing, the container also provides a TLS listener with
+# a self-signed certificate on port 8443.
+#
+# To use custom configuration, use the `server.conf.in` file as template and
+# remove the listeners from [http] and [https] sections. Then provide that file
+# when running the docker container as with `-c` parameter like this:
+#
+#   ```
+#   docker run --rm --name my-spreed-webrtc -p 8080:8080 -p 8443:8443 \
+#       -v `pwd`:/srv/extra -i -t spreed-webrtc \
+#       -c /srv/extra/server.conf
+#   ```
+#
+# And last, this container checks environment variables NEWCERT and NEWSECRETS,
+# on startup. Set those to `1` to regenerate the corresponding values on start.
+# The current certificate and secrets are printed before startup so you can use
+# them easily for other services. Of course, if you want to have persistent cert
+# and secrets, the container needs to be persistent in the first place, so no
+# `--rm` parameter in the example from above in that case.
+#
+
+FROM alpine:3.4
+MAINTAINER Simon Eisenmann <simon@struktur.de>
+
+ENV LC_ALL en_US.UTF-8
+
+# Add glibc.
+ENV GLIBC_PKG_VERSION=2.23-r3
+RUN apk add --update-cache curl ca-certificates openssl && \
```

Comment:
Using `apk add --no-cache` instead of `apk add --update-cache` will stop apk from saving the cache which results in a smaller image (since the cache is not stored in the intermediate layer)